### PR TITLE
Swagger docs for update payments using PUT and PATCH

### PR DESCRIPTION
**Pull request by SharanyaT**

### CODE REVIEW

**Comment by SharanyaT** [comment]
Shoot. My bad - will fix the user id issue now. 
Wrt the patch model scenario - yes, users will be changing 1 or 2 things only, but there's no specific way to figure out before hand what the user will be changing (so as to provide a model acc) which is why I felt it's easier to just state everything and let the user pick whichever he wants (since there's a possibility that they can update details and not know how it looks - since the structure of details is a little complex). If you have a better idea, please tell me and ill make changes :)

**Comment by odoric27** [comment]
@SharanyaT if it not easy/possible to create specific examples while also noting the range of acceptable fields that can be updated, then i'm fine with this.

once the ids are updated this is good to go by me

File: app/documentation/update_payment_patch.yaml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  200:
+    description: Payment updated
+    schema:
+      type: array
+      items:
+        schema:
+          id: Payment
+          properties:
+            id:
+              type: integer
```
Comment (odoric27) [inline app/documentation/update_payment_patch.yaml]: `id` beneath properties should be `payment_id`

File: app/documentation/update_payment_put.yaml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    description: Payment updated
+    schema:
+      type: array
+      items:
+        schema:
+          id: Payment
+          properties:
+            id:
+              type: integer
+              description: id auto-generated by database
```
Comment (odoric27) [inline app/documentation/update_payment_put.yaml]: `id` beneath properties should be `payment_id`

**Pull request merged**